app/models/product.rb

```diff
@@ -204,10 +205,14 @@ def to_s_with_status
 
   def create_default_price_group_products
     PriceGroup.globals.find_each do |price_group|
-      price_group_products.create!(price_group: price_group)
+      price_group_products.create!(price_group:)
     end
   end
 
+  def create_nonbillable_price_policy
+    price_policies << PricePolicy.create_nonbillable_price_policy(self)
```

Comment:
Do you need to shovel here?
```suggestion
    PricePolicy.create_nonbillable_price_policy(self)
```